keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_backbone.py

```diff
@@ -441,7 +454,24 @@ def __new__(
     @classproperty
     def presets(cls):
         """Dictionary of preset names and configurations."""
-        return {}
+        return {
+            "mobilenetv3large": copy.deepcopy(
```

Comment:
We only put in presets with weights into the alias `presets` method because the no weights version is the same as init!

Reply:
See ResNetV2 for an example ([link](https://github.com/keras-team/keras-cv/blob/6c59877637a49a76f9af0e0cefc2573b106c4d19/keras_cv/models/backbones/resnet_v2/resnet_v2_backbone.py#L520)).